src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const NotPositive = a <= 0 || b <= 0 || c <= 0;
+    const isNotTriangle = a + b <= c || a + c <= b || b + c <= a;
+
+    if (NotPositive || isNotTriangle) {
+      throw new Error('incorrect value');
+    }
+    this.color = color;
```

Comment:
As I know, we don't need to create here `this.color = color;` because in 11line, TypeScript do it instead of us
```suggestion

```
It's repeated again in the code belowe👀👀

Reply:
> As I know, we don't need to create here `this.color = color;` because in 11line, TypeScript do it instead of us
> 
> It's repeated again in the code belowe👀👀

Good point